### DISCUSSION TITLE
Synchronization with AWS S3

### DISCUSSION THREAD

**Discussion by tangytang** [iTwin/community#65] [Q&A] [answered]
We have a number of Revit & IFC stored in an AWS S3 bucket.

## Outcomes
- User is able to upload Revit files via a web browser
- Application is able to automatically render uploaded files in web-viewer
- User is able to roll back and visually see (previous) versions of Revit files

## Envisioned workflow is to:
1) Get files from S3 bucket
2) Run Synchronization API on files
3) Display as a model using iTwin/web-viewer

### Questions
- Does the user require a Bentley IMS identity to be able to access the models?

Is the above-mentioned workflow possible?

**Comment by Aidas-Vaiksnoras** [comment, marked as answer]
Hello,
the workflow above is possible with Synchronization Manifest API. You can follow this tutorial https://developer.bentley.com/tutorials/synchronization-blob-tutorial/ Only difference will be that the files are stored in AWS S3 bucket, so in Step 6 the URL should be a preauthenticated URL from AWS S3 blob.
